com.civicrm.hrjobroles/js/test/controllers/job-roles.controller.spec.js

```diff
@@ -75,18 +75,6 @@ define([
               'is_reserved': '0',
               'is_active': '1'
             },
-            {
```

Comment:
Why was it moved?

Reply:
I want to make it consistent with other sample data (other than CostCentreList)